components/definition/cellRules/ImportCellParams.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import Card from 'material-ui/Card/Card';
+import Checkbox from 'material-ui/Checkbox';
+import TextField from 'material-ui/TextField';
+import Dialog from 'material-ui/Dialog/Dialog';
+import CardText from 'material-ui/Card/CardText';
+import FlatButton from 'material-ui/FlatButton/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton/RaisedButton';
+import Utils from '../../../Utils';
+import NetPyNEField from '../../general/NetPyNEField';
+
+var PythonControlledCapability = require('../../../../../js/communication/geppettoJupyter/PythonControlledCapability');
+
+export default class ImportCellParams extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      label: props.name,
+      conds: {},
+      fileName: '',
+      cellName: '',
+      modFolder: '',
+      compileMod: false,
+      importSynMechs: false,
+    };
+    this.updateCheck = this.updateCheck.bind(this);
+    this.performAction = this.performAction.bind(this);
+  };
+  
+  updateCheck(name) {
+    this.setState((oldState) => {
+      return {
+        [name]: !oldState[name],
+      };
+    });
+  };
+
+  performAction = () => {
+    // Show spinner
+    var message = GEPPETTO.Resources.IMPORTING_MODEL;
+    GEPPETTO.trigger(GEPPETTO.Events.Show_spinner, message);
+
+    // Import template
+    var action = 'netpyne_geppetto.importCellTemplate';
```

Comment:
Same here, no need for this variable. Just use it straight and the code will be easier to read